src/legacy/index.ts

```diff
@@ -1,6 +1,6 @@
 import * as crypto from 'crypto';
 import * as types from '../core/types';
-import { DepGraphBuilder } from '../core/builder';
+import { DepGraphBuilder } from '..';
```

Comment:
Unneeded change. Please restore this line - the way it's importing right now creates more static coupling than before